docs/userdocs/k8s/helm.md

```diff
@@ -20,31 +20,91 @@ under the License.
 
 # Deploy Submarine On K8s
 
-## Deploy Submarine using Helm Chart (Recommended)
+## Deploy Submarine Using Helm Chart (Recommended)
 
 Submarine's Helm Chart will not only deploy Submarine Server, but also deploys TF Operator / PyTorch Operator (which will be used by Submarine Server to run TF/PyTorch jobs on K8s).
 
-### Create images
-submarine server
+
+### Install Helm
+
+Helm v3 is minimum requirement.
+See here for installation: https://helm.sh/docs/intro/install/
+
+### Install Submarine
+
+The Submarine helm charts is released with the source code for now.
+Please go to `http://submarine.apache.org/download.html` to download
+
+- Install Helm charts from source code
 ```bash
-./dev-support/docker-images/submarine/build.sh
+cd <PathTo>/submarine
+helm install submarine ./helm-charts/submarine
 ```
+This will install submarine in the "default" namespace.
+The images are from Docker hub `apache/submarine`. See `./helm-charts/submarine/values.yaml` for more details
 
-submarine database
+If we'd like use a different namespace like "submarine"
 ```bash
-./dev-support/docker-images/database/build.sh
+kubectl create namespace submarine
+helm install submarine ./helm-charts/submarine -n submarine
 ```
 
-### install helm
-For more info see https://helm.sh/docs/intro/install/
+> Note that if you encounter below issue when installation:
+```bash
+Error: rendered manifests contain a resource that already exists.
+Unable to continue with install: existing resource conflict: namespace: , name: podgroups.scheduling.incubator.k8s.io, existing_kind: apiextensions.k8s.io/v1beta1, Kind=CustomResourceDefinition, new_kind: apiextensions.k8s.io/v1beta1, Kind=CustomResourceDefinition
+```
+It might be caused by the previous installed submarine charts. Fix it by running:
+```bash
+kubectl delete crd/tfjobs.kubeflow.org && kubectl delete crd/podgroups.scheduling.incubator.k8s.io && kubectl delete crd/pytorchjobs.kubeflow.org
+```
+
+- Verify installation
 
-### Deploy Submarine Server, mysql
-You can modify some settings in ./helm-charts/submarine/values.yaml
+Once you got it installed, check with below commands and you should see similar outputs:
 ```bash
-helm install submarine ./helm-charts/submarine
+kubectl get pods
+```
+
+```bash
+NAME                                 READY     STATUS    RESTARTS   AGE
+pytorch-operator-54854bf847-x65nk    1/1       Running   0          5m
+submarine-database-5f74f747d-dzmf6   1/1       Running   0          5m
+submarine-server-6f449bc967-cqkkv    1/1       Running   0          5m
+tf-job-operator-c9cd7ccbd-4dzcs      1/1       Running   0          5m
 ```
 
-### Delete deployment
+### Enable local access to Submarine Server
```

Comment:
it's better to add an explanation that: Submarine server by default exposed 8080 port within K8s cluster. To access the server from outside of the cluster, we need to expose 8080. We can either use port-forward, or use `Ingress`, here is an example of port-forward.

Reply:
Good idea.